src/blocks/form/components/edit.js

```diff
@@ -0,0 +1,403 @@
+		const emails = to.trim();
+		if ( emails.length === 0 ) {
+			this.setState( { toError: null } );
+			this.props.setAttributes( { to } );
+			return;
+		}
+
+		this.setState( { toError: null } );
+		this.props.setAttributes( { to } );
+	}
+
+	onChangeSubmit( submitButtonText ) {
+		this.props.setAttributes( { submitButtonText } );
+	}
+
+	onFormSettingsSet( event ) {
+		event.preventDefault();
+		if ( this.state.toError ) {
+			// don't submit the form if there are errors.
+			return;
+		}
+		this.props.setAttributes( { hasFormSettingsSet: 'yes' } );
+	}
+
+	getfieldEmailError( errors ) {
+		if ( errors ) {
+			if ( errors.length === 1 ) {
+				if ( errors[ 0 ] && errors[ 0 ].email ) {
+					return sprintf( __( '%s is not a valid email address.' ), errors[ 0 ].email );
+				}
+				return errors[ 0 ];
+			}
+
+			if ( errors.length === 2 ) {
+				return sprintf(
+					__( '%s and %s are not a valid email address.' ),
+					errors[ 0 ].email,
+					errors[ 1 ].email
+				);
+			}
+			const inValidEmails = errors.map( error => error.email );
+			return sprintf(
+				__( '%s are not a valid email address.' ),
+				inValidEmails.join( ', ' )
+			);
+		}
+		return null;
+	}
+
+	preventEnterSubmittion( event ) {
+		if ( event.key === 'Enter' ) {
+			event.preventDefault();
+			event.stopPropagation();
+		}
+	}
+
+	renderToAndSubjectFields() {
+		const fieldEmailError = this.state.toError;
+		const { instanceId, attributes } = this.props;
+		const { subject, to } = attributes;
+		return (
+			<Fragment>
+				<TextControl
+					aria-describedby={ `contact-form-${ instanceId }-email-${
+						this.hasEmailError() ? 'error' : 'help'
+					}` }
+					label={ __( 'Email address' ) }
+					placeholder={ __( 'name@example.com' ) }
+					onKeyDown={ this.preventEnterSubmittion }
+					value={ ( to || '' === to ) ? to : coblocksBlockData.form.adminEmail }
+					onBlur={ this.onBlurTo }
+					onChange={ this.onChangeTo }
+				/>
+				<HelpMessage isError id={ `contact-form-${ instanceId }-email-error` }>
+					{ this.getfieldEmailError( fieldEmailError ) }
+				</HelpMessage>
+				<HelpMessage id={ `contact-form-${ instanceId }-email-help` }>
+					{ this.getEmailHelpMessage() }
+				</HelpMessage>
+
+				<TextControl
+					label={ __( 'Email subject line' ) }
+					value={ ( subject || '' === subject ) ? subject : coblocksBlockData.form.emailSubject }
+					placeholder={ __( "Let's work together" ) }
+					onChange={ this.onChangeSubject }
+				/>
+			</Fragment>
+		);
+	}
+
+	hasEmailError() {
+		const fieldEmailError = this.state.toError;
+		return fieldEmailError && fieldEmailError.length > 0;
+	}
+
+	render() {
+
+		const {
+			attributes,
+			className,
+			isSelected,
+			setAttributes,
+			settings,
+			isLargeViewport,
+			isRTL,
+			toggleSelection,
+			maxWidth,
+		} = this.props;
+
+		const {
+			align,
+			alt,
+			height,
+			id,
+			url,
+			width,
+			caption,
+		} = attributes;
+
+		return (
+			<Fragment>
+				<InspectorControls>
```

Comment:
We should pull the InspectorControls into it's own inspector.js file. Do you see foresee an issue with that @EvanHerman?

Reply:
@richtabor No that shouldn't be an issue 👍